### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Containers_Create Blob Container_ColorContrast : The Color contrast ratio of blob container edit field focus border is less than 3:1

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3460]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.11;#Win32;#A11ySev2;#Benchmark;#ColorContrast;#FTP;

### **Environment Details:**

Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Edit box will appear to give name to new Blob container.
6. Open AI4D and Check the color contrast of focus border of edit box.

### **Actual:**
The color contrast ratio of blob container edit field focus border is less than 3:1

### **Expected:**
The color contrast ratio of blob container edit field focus border should be greater than 3:1.

### **User Impact:**
Low vision users may not able to see the control has keyboard focus.

### **Attachment**
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prraha_microsoft_com/EesSoQuJFEJBt6RahCTmxdcBd2p7jIika3L4c_YMcWIgxQ?e=F4O64i

**Comment by pranshu512** [comment]
I verified this issue and found that it is not reproducing 
hence,closing this bug
![image](https://user-images.githubusercontent.com/64009219/96445516-a9692280-122d-11eb-97f0-622cd862fdd6.png)

**Comment by pranshu512** [comment]
I verified this issue and found that it is not reproducing
hence,closing this bug

![image](https://user-images.githubusercontent.com/64009219/96561491-433bd880-12dd-11eb-8977-306bed680f16.png)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
